ext/native-decls/GetEntityFromStateBagName.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: CFX
+apiset: shared 
+---
+## GET_ENTITY_FROM_STATE_BAG_NAME
+
+```c
+Entity GET_ENTITY_FROM_STATE_BAG_NAME(char* bagName);
+```
+
```

Comment:
A description could help, perhaps as follows:

> `Returns the entity handle for the specified state bag name. For use with [ADD\_STATE\_BAG\_CHANGE\_HANDLER](?_0x5BA35AAF).`

Adding an example, perhaps, too?

Reply:
Maybe it would make more sense to add an example on AddStateBagChangeHandler and reference it here?